assets/src/components/v2/normal_header.tsx

```diff
@@ -28,14 +28,6 @@ const ICON_TO_SRC: Record<Icon, string> = {
   logo_negative: "logo-black.svg",
 };
 
-const abbreviateText = (text: string) => {
-  if (text === "Government Center") {
-    return "Government Ctr";
```

Comment:
Note: This one-off bit of abbreviation logic was only used on Pre-Fares. Since these [now support](https://github.com/mbta/screens/pull/2575) configuring arbitrary text as the header, we don't need this anymore.